docs/tutorial/features/books.feature

```diff
@@ -1,10 +1,12 @@
+# Feature files represent `Application under test` functional capabilities
+# in form of acceptance test with representative examples
 Feature: Library book searches and book delivery
 Scenario: The catalog can be searched by author name.
     Given these books in the catalog
     | Author          | Title                       |
     | Stephen King    | The Shining                 |
     | James Baldwin   | If Beale Street Could Talk  |
-    When a name search is performed for Stephen
+    When a name search is performed for "Stephen"
```

Comment:
Personally I wouldn't put this in quotation marks. It certainly isn't wrong to use quotation marks, but the step definition doesn't need them, and this forces the requirements writer to remember them. What are the arguments for using quotation marks?

Reply:
IMO it's a question of taste, I usually add them to show that this step is parameterized and could be used with some other value.

Reply:
Removed quotations